Sources/Rules.swift

```diff
@@ -5144,6 +5144,36 @@ public struct _FormatRules {
         }
     }
 
+    /// Remove unused private and fileprivate declarations
+    public let unusedPrivateDeclaration = FormatRule(
+        help: "Remove unused private and fileprivate declarations."
```

Comment:
Seems like a good idea for this to be disabled by default

```suggestion
        help: "Remove unused private and fileprivate declarations.",
        disabledByDefault: true
```

Reply:
what do you think @nicklockwood?